data/wikis.json

```diff
@@ -954,11 +1198,58 @@
 				"id": "32794352",
 				"name": "FANDOMbot"
 			},
+			{
+				"id": "32997079",
+				"name": "Savaphoong"
+			},
+			{
+				"id": "34293559",
+				"name": "Your Werewolf Jesus Savior"
+			},
 			{
 				"id": "34480145",
 				"name": "I make userpages"
+			},
+			{
+				"id": "38461327",
+				"name": "RGTbot"
 			}
 		],
 		"domain": "psychology.wikia.org"
+	},
+	{
+		"domain": "hitchwiki.org/en",
+		"url": "https://hitchwiki.org/en",
+		"data": "hitchwiki.org_en.csv",
```

Comment:
This file (`hitchwiki.org_en.csv`) is missing in the PR.

Reply:
Solved in https://github.com/Grasia/WikiChron/pull/89/commits/a5d3072e922849e36a72feb7403d52db1ac8359e